stats/cloud/data.go

```diff
@@ -110,6 +110,29 @@ type SampleDataMap struct {
 // NewSampleFromTrail just creates a ready-to-send Sample instance
 // directly from a httpext.Trail.
 func NewSampleFromTrail(trail *httpext.Trail) *Sample {
+	if trail.Failed.Valid {
+		return &Sample{
```

Comment:
Instead of copy-pasting the whole `&Sample{}` definition based on `trail.Failed`, can you please only change the `Values` map?

Reply:
I specifically did this so I don't expand every time as you pointed out in https://github.com/loadimpact/k6/pull/1856#discussion_r584496342

Reply:
Yeah, I get that, but you can only copy the `Values` map init, not the whole `Sample` struct

Reply:
done